common/web/types/src/ldml-keyboard/pattern-parser.ts

```diff
@@ -64,6 +64,9 @@ export class MarkerParser {
   public static readonly MAX_MARKER_INDEX = constants.marker_max_index;
   /** Max count of markers */
   public static readonly MAX_MARKER_COUNT = constants.marker_max_count;
+  /** Expression that matches any marker */
+  public static readonly ANY_MARKER_MATCH =
+    this.SENTINEL + this.MARKER_CODE + `[\\u0001-\\u${this.MAX_MARKER_INDEX.toString(16)}]`;
```

Comment:
At least the end of the range is dynamic. Or is it worth importing left-pad here for the beginning?

Reply:
I'm not sure I understand the comments here?

Reply:
@mcdurdin sorry, it was obscure.

I was joking about whether I should `require('left-pad')`  (of infamy https://www.theregister.com/2016/03/23/npm_left_pad_chaos/ ) in order to convert `1` into `0001`.

Reply:
Surely we should define `MIN_MARKER_INDEX` as 1. Then you might get your chance to left pad. But with `padStart` please 🤣